packages/core/src/components/toast/toast.md

```diff
@@ -58,51 +51,71 @@ OurToaster.update(key, { message: "Still toasted!" });
     enable `autoFocus` for a `Toaster` via a prop, if desired.
 </div>
 
-@### Static method
+@interface IToaster
```

Comment:
I think this should go inside the "Static usage" section, after the first bit of prose there. Prefer reading prose first, detailed props tables later